js/components/home.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from "react";
+
+export const Home = (): ReactElement => {
+  return (
+    <div className="text-3xl">
+      🪐
+      <span className="text-[#ED8B00]">O</span>
+      <span className="text-[#DA291C]">r</span>
+      <span className="text-[#003DA5]">b</span>it
```

Comment:
**praise:** beautiful.

Reply:
This was all @skyqrose, I just brought it over to the react side. I agree, WD Sky!